sensor/extrinsic_reflector_based_calibrator/src/extrinsic_reflector_based_calibrator.cpp

```diff
@@ -1474,3 +1634,67 @@ void ExtrinsicReflectorBasedCalibrator::visualizationMarkers(
 
   tracking_markers_pub_->publish(tracking_marker_array);
 }
+
+void ExtrinsicReflectorBasedCalibrator::deleteTrackMarkers()
+{
+  visualization_msgs::msg::MarkerArray tracking_marker_array;
+  visualization_msgs::msg::Marker marker;
+  auto deleted_id_start = converged_tracks_.size() * 8;
```

Comment:
Thanks for implementing what we discussed the other day.
I can see what you are trying to do here, but just putting an 8 here looks like a magic number.
It is a good practice to add a constexpr in capitals somewhere with a descriptive name

Reply:
Now it is using `static constexpr int MARKER_SIZE_PER_TRACK = 8;`
commit https://github.com/tier4/CalibrationTools/pull/145/commits/82d940da6f5dfdebeea22fd0649d4526e63a8c7f